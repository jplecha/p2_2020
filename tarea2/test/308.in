insertarAlFinal (2,8.4)
insertarAlFinal (7,4.6)
insertarAlFinal (4,5.5)
insertarAlFinal (6,7.2)
insertarAlFinal (7,4.3)
insertarAlFinal (8,1.8)
insertarAlFinal (1,1.9)
insertarAlFinal (7,8.4)
insertarAlFinal (3,3.8)
finalCadena
cambiarTodos
imprimirCadena
finalCadena
finalCadena
localizadorEnCadena
insertarSegmentoDespues (6,2.8) (6,7.8) (4,4.4) (4,2.7) (9,9.3) (3,1.9) (1,5.1)
ordenar
precedeEnCadena
finalCadena
finalCadena
precedeEnCadena
esLocalizador
estaOrdenadaPorNaturales
finalCadena
finalCadena
