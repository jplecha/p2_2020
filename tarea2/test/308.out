==12769== Conditional jump or move depends on uninitialised value(s)
==12769==    at 0x4039E6: cambiarTodos(unsigned int, unsigned int, repCadena*) (usoTads.cpp:148)
==12769==    by 0x4022C4: main (principal.cpp:333)
==12769== 
==12769== Conditional jump or move depends on uninitialised value(s)
==12769==    at 0x4012F4: leerCadena() (principal.cpp:79)
==12769==    by 0x401B06: main (principal.cpp:237)
==12769== 
principal: principal.cpp:327: int main(): Assertion `! hayNatsRepetidos(cad)' failed.
1>Insertado al final.
2>Insertado al final.
3>Insertado al final.
4>Insertado al final.
5>Insertado al final.
6>Insertado al final.
7>Insertado al final.
8>Insertado al final.
9>Insertado al final.
10>loc al final.
11>Cambiados.
12>loc al final.
13>loc al final.
14>loc pertenece a cad.
15>Segmento insertado despuÃ©s de loc.
16>timeout: the monitored command dumped core
